CMakeLists.txt

```diff
@@ -39,7 +39,7 @@ add_subdirectory(AssetLoader)
 add_subdirectory(Imgui)
 add_subdirectory(NativeApp)
 
-if((PLATFORM_WIN32 OR PLATFORM_LINUX OR PLATFORM_MACOS) AND GL_SUPPORTED)
+if((PLATFORM_WIN32 OR PLATFORM_LINUX OR PLATFORM_MACOS OR PLATFORM_EMSCRIPTEN) AND GL_SUPPORTED)
```

Comment:
On Emscripten, `GL_SUPPORTED` is `False`, `GLES_SUPPORTED` is `True`.
This is a standalone converter, it is supposed to run on desktop platforms only.